mlflow/pyfunc/__init__.py

```diff
@@ -724,6 +727,85 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def _download_model_conda_env(model_uri):
+    conda_yml_file_name = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)[ENV]
+    return _download_artifact_from_uri(os.path.join(model_uri, conda_yml_file_name))
+
+
+def get_model_dependencies(model_uri, format="pip"):  # pylint: disable=redefined-builtin
+    """
+    Given a model URL, and format ("pip" or "conda", default value "pip"),
+    return the downloaded dependency file path.
+    If "pip" format specified but model does not have "requirements.txt" file,
+    fallback to parse the pip section of the model's "conda.yaml" and ignore other
+    non-pip dependencies in the "conda.yaml" file.
+    """
+    req_file_uri = os.path.join(model_uri, _REQUIREMENTS_FILE_NAME)
+
+    if format == "pip":
+        try:
+            pip_file_path = _download_artifact_from_uri(req_file_uri)
```

Comment:
```suggestion
            return _download_artifact_from_uri(req_file_uri)
```

Can we exit-fast if this `_download_artifact_from_uri` call succeeds?

Reply:
we need to also print 
```
        if is_in_databricks_runtime():
            _logger.info(
                "To install these model dependencies in your Databricks notebook, run the "
                f"following command: '%pip install -r {pip_file_path}'."
            )
``
before return.

Reply:
I missed that line.